packages/titanium-image-input/src/image-cropper-dialog.ts

```diff
@@ -17,6 +17,7 @@ export class ImageCropperDialogElement extends LitElement {
   @property({ type: Object }) file: File | null = null;
   @property({ type: String }) fileName: string = '';
   @property({ type: String }) previewDataUrl: string | null = null;
+  @property({ type: Boolean }) circle: boolean = false;
```

Comment:
It would be nice if we could perhaps extend Cropper.Options.  to ```Cropper.Options & { shape: ' square'|'circle'}```.

Reply:
I like that. Done